input/MergeStars/run.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+# this script should be executed from the repository's root directory
+
+# these tests should be integrated with the rest of the tests.
+
+testPrefix="./input/MergeStars"
+
+mkdir -p test/MethodMergeStars
+
+echo "Checking that the python installation roughly matches test requirements"
+# One last set of crude checks to make sure that environment is not obviously
+# unfit for running the tests
+USING_PY3=$(python -c "exec(\"import sys\nprint(int(sys.version_info.major==3))\")")
+if [[ "$USING_PY3" != "1" ]]; then
+    echo "the 'python' command does not launch a version of python 3."
+    echo "this is required for running MergeStars tests."
+    exit 1
+fi
+
+YT_INSTALLED=$(pip list | grep "^yt" | wc -l )
+if [[ "$YT_INSTALLED" == "0" ]]; then
+    echo "yt is not installed. It is required for running MergeStars tests"
+    exit 1
+fi
+
+MPI4PY_INSTALLED=$(pip list | grep "^mpi4py" | wc -l )
```

Comment:
I just checked: the wallclock time with MPI is 1m45s. Without: 2m35s. So not a huge speed-up. I used MPI originally because the tests were generating (and the processing)100 snapshots rather than 10 as they are now.

Reply:
So just to clarify, do you recommend not using mpi4py? I suppose it might not be strictly necessary, but I don't see the harm in using it. I don't expect it to be too much work to remove it though.

Reply:
My thinking is as follows:
- `mpi4py` requires MPI to be installed to run tests.
- This could complicate things for new users/developers who want to run the test suite on a local machine, where MPI is not already installed. Installing MPI is fairly non-trivial if you don't have root access to the machine (e.g. I'm thinking primarily of grad students)

At the same time, this is a fairly niche (hypothetical) concern, and this is not a hill I want to die on. So I'll leave the choice up to you.

Reply:
I've just implemented this change.